src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -350,7 +363,7 @@ private static IEdmStructuredTypeReference GetResourceType(IEdmTypeReference fee
             throw new SerializationException(message);
         }
 
-        private static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, int pageSize)
+        internal static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, int pageSize)
```

Comment:
>internal [](start = 8, length = 8)

why make this internal?

Reply:
I wanted to share this for DeltaFeedSerializer as I found repetition in code. Ideally, this should have been on a shared class but I thought this may be fine for the time being. 

---
In reply to: [287399610](https://github.com/OData/WebApi/pull/1724#discussion_r287399610) [](ancestors = 287399610)